math/src/field/fields/fft_friendly/babybear.rs

```diff
@@ -98,4 +116,20 @@ mod test_babybear_31_bytes_ops {
         let from_bytes = FieldElement::<Babybear31PrimeField>::from_bytes_be(&bytes).unwrap();
         assert_eq!(element, from_bytes);
     }
+
```

Comment:
There should be a test using FFT to show that everything's working properly. Also, use cargo fmt to help with the code formatting

Reply:
should the FFT test be added inside babybear.rs? or inside polynomial.rs like was done for `Stark252PrimeField`?

Reply:
You can add it inside babybear, but under the cfg[test], so it's only part of that module for testing purposes.